web-client/src/modules/create/components/PostSummaryStep.tsx

```diff
@@ -28,17 +27,17 @@ const PostSummary: React.FC<PostSummaryProps> = ({
   const [showConfirmationPage, setShowConfirmationPage] = useState(false);
   const history = useHistory();
 
-  const handleSubmit = () => setShowConfirmationPage(true);
+  const handleSubmit = () => {
+    submitPost();
```

Comment:
Should we await this before showing the modal? What should we do if this fails for some reason?

Reply:
Point well taken.   I thought the delay time for the interaction with the modal would be sufficient for the new request to propagate to the firebase database.  But I will try to take advantage of the promise.

Reply:
@alexjball fixed